build-aux/libreoffice/custom-profile.xcu

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<oor:items xmlns:oor="http://openoffice.org/2001/registry" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<item oor:path="/org.openoffice.Office.Common/Filter/PDF/Export"><prop oor:name="ExportLinksRelativeFsys" oor:op="fuse"><value>false</value></prop></item>
```

Comment:
This is magic. Could you add one or more comments in it explaining what it does and linking to relevant references?

Reply:
hehe, indeed it is magic, truth being told I found this after lots of google searches when I finally saw a post mentioning we could disable using relative urls by using `File > Export As > Export as PDF`. Then I tried it and checked my saved profile to identify what option was set...

The closest doc I found is https://help.libreoffice.org/Common/Export_as_PDF#Export_URLs_relative_to_file_system which unfortunately doesn't mention the option.

If you still think it is worth I can add a comment and link to this page:
```
<!-- Disable option to export links relative to filesystem - https://help.libreoffice.org/Common/Export_as_PDF#Export_URLs_relative_to_file_system -->
```

Reply:
Yeah, or something like:
```
<!-- Disable option to export links relative to filesystem - https://help.libreoffice.org/Common/Export_as_PDF#Export_URLs_relative_to_file_system
     Found by using File -> Export As -> Export as PDF -> Links and checking the changes to the LibreOffice saved profile. -->
```

Reply:
Done.